Azure-Samples/modern-data-warehouse-dataops
# ydaponte/584 data retention

# Type of PR

- Code changes
- Test changes

## Purpose
- Adding Data Retention Pipeline to the Sample

## Does this introduce a breaking change? If yes, details on what can break
- No.

## Author pre-publish checklist

## Validation steps
- Deploy sample
- Run Main pipeline
- Run retention pipeline and verify delete changes to folders 2019 and 2020 and on folder 2021/01 check changes with the deletion and vacuum
- Check changes on configuration file (no longer references entries from years 2019 and 2020 - as the retention period was defined to 2 years)

## Issues Closed or Referenced
- Closes #584
